user-operator/src/main/java/io/strimzi/operator/user/UserOperatorConfig.java

```diff
@@ -46,512 +58,381 @@ public class UserOperatorConfig {
     static final String STRIMZI_BATCH_MAXIMUM_BLOCK_TIME_MS = "STRIMZI_BATCH_MAXIMUM_BLOCK_TIME_MS";
     static final String STRIMZI_USER_OPERATIONS_THREAD_POOL_SIZE = "STRIMZI_USER_OPERATIONS_THREAD_POOL_SIZE";
 
-    // Default values
-    static final long DEFAULT_FULL_RECONCILIATION_INTERVAL_MS = 120_000;
-    static final String DEFAULT_KAFKA_BOOTSTRAP_SERVERS = "localhost:9091";
+    /**
+     * Namespace in which the operator will run and create resources
+     */
+    public static final ConfigParameter<String> NAMESPACE = new ConfigParameter<>(STRIMZI_NAMESPACE, NON_EMPTY_STRING, "", true, CONFIG_VALUES);
```

Comment:
Are the names of the env vars used also somewhere else? If not, wouldn't the resulting code be more readable if the strings were just inlined and the constants removed? E.g.

```suggestion
    public static final ConfigParameter<String> NAMESPACE = new ConfigParameter<>("STRIMZI_NAMESPACE", NON_EMPTY_STRING, "", true, CONFIG_VALUES);
```

Reply:
You did that for the defaults. Can't it be done similarly? Or use the Builder instead of the maps?

Reply:
But the tests where the constants are being used are testing if the configuration are coming our correctly from the map. For eg. in the `UserOperatorConfigTest`.

Reply:
```
        ENV_VARS.put(UserOperatorConfig.NAMESPACE.key(), "namespace");
        ENV_VARS.put(UserOperatorConfig.RECONCILIATION_INTERVAL_MS.key(), "30000");
```

Reply:
I can do something like this if its fine?

Reply:
Yeah, I think that is good.

Reply:
Great, I will make these changes then. Thanks Jakub

Reply:
Done